package-overrides/npm/Leaflet.vector-markers@0.0.6.json

```diff
@@ -1,4 +1,6 @@
 {
-  "dependencies": {},
+  "dependencies": {
+    "leaflet": "^1.0.0"
```

Comment:
Thanks for the PR. This looks great, but for maximum compatibility the minimum version should be `"^0.7.0"` to match the requirements specified here https://github.com/hiasinho/Leaflet.vector-markers/blob/master/package.json#L33

That is of course there is an incompatibility with this version that you are aware of.